RetailCoder.VBE/Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoring.cs

```diff
@@ -135,9 +135,8 @@ private void UpdateOtherModules()
                 oldSelection = module.GetQualifiedSelection();
             }
 
-            var newTarget = _state.AllUserDeclarations.FirstOrDefault(
+            var newTarget = _state.DeclarationFinder.MatchName(_target.IdentifierName).FirstOrDefault(
```

Comment:
I've been working on moving the inspections over. Some of them have become parse tree inspections (no use of declarations at all). Others have been moved to use the DeclarationFinder. I think I have over half of them off it now.